src/main/java/net/minecraftforge/fmp/capabilities/CapabilityWrapperItemHandler.java

```diff
@@ -0,0 +1,124 @@
+package net.minecraftforge.fmp.capabilities;
+
+import java.util.Collection;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.items.CapabilityItemHandler;
+import net.minecraftforge.items.IItemHandler;
+import net.minecraftforge.items.IItemHandlerModifiable;
+
+public class CapabilityWrapperItemHandler implements ICapabilityWrapper<IItemHandler>
+{
```

Comment:
This is meant to take multiple implementations of the same capability and wrap them into just one object which can be returned when that cap is requested on the side of the multipart TE. This way other developers that just want to access the parts' inventory, for example, can insert items normally without having to adapt their code to iterate through all of them.